group-aws/integration.md

```diff
@@ -0,0 +1,34 @@
+# Integration Testing
+
+There are very few integration tests for EC2. Those that
+exist are in test/integration/targets
+
+## Expected test criteria
+
+* Resource creation
+* Resource creation again
+* Resource creation under check mode
+* Resource modification
+* Resource deletion
+
+When adding new features, those features should have tests.
+
+# Github PRs
+There are some PRs with tests:
+
+| Module               | PR                                                     |
+|----------------------|--------------------------------------------------------|
+| lambda               | [24951](https://github.com/ansible/ansible/pull/24951) |
+| rds                  | [25646](https://github.com/ansible/ansible/pull/25646) |
+| rds_param_group      | [25345](https://github.com/ansible/ansible/pull/25345) |
+
+# State of the codebase
+
+Modules with non-trivial tests:
```

Comment:
There are some modules that have targets under test/integration/targets but tasks/main.yml is empty. 

However, rather than define those as trivial, I should just say 'Existing modules with tests'